repository.rdf

```diff
@@ -417,16 +336,6 @@
 
 
 
-    <!-- http://fedora.info/definitions/v4/repository#Configuration -->
-
-    <owl:Class rdf:about="http://fedora.info/definitions/v4/repository#Configuration">
```

Comment:
Gotcha, so then...is there a way to actually use the versioning of the ontology? Or is it more just for a historical perspective?